app/views/guides/pages/getting-started/handling-events.haml

```diff
@@ -19,10 +19,14 @@
 
   An event attribute is always starts with
   %code on
-  followed by the name of the event (capitalized), for example:
+  followed by the name of the event (
+  %a(href="/guide/reference/javascript-interop/decode-expression")
```

Comment:
Why does this link to the decode expression?

Reply:
Was supposed to link to the camelCase article on wikipedia, fixed